components/organisms/Feedback.js

```diff
@@ -0,0 +1,188 @@
+import PropTypes from "prop-types";
+import { useTranslation } from "next-i18next";
+import { useState } from "react";
+import { ActionButton } from "../atoms/ActionButton";
+
+/**
+ * Feedback component
+ */
+export function Feedback(props) {
+  const [submitted, setSubmitted] = useState(false);
+  const { t } = useTranslation();
+
+  let onSubmitHandler = (e) => {
+    // prevent default behaviour of form
+    e.preventDefault();
+    // create FormData object from form
+    const formData = new FormData(e.target);
+    // create URLSearchParams object from FormData object
+    // this will be used to create url encoded string of names and values of the form fields
+    const urlEncoded = new URLSearchParams(formData);
+    // call feedback API route
+    fetch("/api/feedback", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/x-www-form-urlencoded;charset=UTF-8",
+      },
+      body: urlEncoded.toString(),
+    }).catch((e) => {
+      // handle error if fetch fails
+      // fetch only fails if there is no internet connection not for the actual
+      // request so there is nothing really to do here other than to log it
+      console.log(e);
+    });
+
+    setSubmitted(true);
+  };
+
+  //Function for closing feedback
+  function closeFeedback() {
+    const feedbackButton = document.getElementById("feedbackButton");
+    const feedbackDropdown = document.getElementById("feedbackDropdown");
+    const caret = document.getElementById("caret");
+
+    feedbackDropdown.classList.toggle("active");
+    feedbackButton.getAttribute("aria-expanded") === "true"
+      ? feedbackButton.setAttribute("aria-expanded", false)
+      : feedbackButton.setAttribute("aria-expanded", true);
+
+    feedbackButton.getAttribute("aria-expanded") === "true"
+      ? (caret.innerHTML = "&#9660;")
+      : (caret.innerHTML = "&#9656;");
+  }
```

Comment:
Similar to the submit feedback function, I think this should be handled using React state.